fuel-vm/src/interpreter/executors/instruction.rs

```diff
@@ -26,7 +26,7 @@ where
             .ok_or(InterpreterError::Panic(PanicReason::MemoryOverflow))?;
 
         // Store the expected `$pc` after executing `hi`
-        let pc = self.registers[REG_PC] + core::mem::size_of::<RawInstruction>() as Word;
+        let pc = self.registers[REG_PC] + Instruction::SIZE as Word;
```

Comment:
We might want to be careful here - `RawInstruction` is a type alias for `u32`, and doesn't necessarily *need* to match the `Instruction` size.

It just so happens that I *have* made sure that `Instruction` itself *is* the same size as the `RawInstruction` (as it doesn't require any more space for the same information) with an included test in `fuel-asm/src/lib.rs`, so this is technically safe.

That said, we might be better off naming this const `RAW_SIZE` or having a separate `fuel_asm::RAW_INSTRUCTION_SIZE` or whatever to clarify that we're referring to the `u32` repr? If not, we might want to ensure that `Instruction`'s size always matches `RawInstruction`'s size. We do have a test for this, but it might be worth adding a comment to explain why it's important that the sizes match.

Reply:
I've added a comment to the relevant test explaining the importance of this, that should be enough https://github.com/FuelLabs/fuel-vm/pull/283/commits/dd41273fc05c0effc80e6959089681e4125f6072.